src/Http/Controllers/Corporation/ViewController.php

```diff
@@ -188,6 +188,70 @@ public function getStarbases($corporation_id)
     }
 
     /**
+<<<<<<< HEAD
+     * @param $corporation_id
+     *
+     * @return \Illuminate\Contracts\View\Factory|\Illuminate\View\View
+     */
+    public function getStarbaseSilos($corporation_id)
+    {
+
+        // not pretty but required
+        ini_set('max_execution_time', 12000);
```

Comment:
Hmm, yes we can certainly try. It's likely never going to be *that speedy though. We should at the very least re-set the max_execution_time after this which looking at my code I haven't.

There's a couple of things you've spotted that I should of, thank you.

I'll get to updating these this week and see if we can make this sleek enough for merging.

Thanks Leonjza.